tracer/src/Datadog.Trace/Ci/Tagging/TestSpanTags.cs

```diff
@@ -97,4 +97,19 @@ public TestSpanTags(TestSuiteSpanTags suiteTags, string testName)
 
     [Tag(EarlyFlakeDetectionTags.TestIsRetry)]
     public string EarlyFlakeDetectionTestIsRetry { get; set; }
+
+    [Tag(BrowserTags.BrowserDriver)]
```

Comment:
Is there really any value to adding these? The `TestSpanTags` is getting huge, and IIRC, you pretty much only use the slow-path for setting these, so it feels like you're getting the worst of all worlds in terms of memory usage/perf?

Reply:
Slow path is not so slow, we actually have a fast path in the `GetTag` `SetTag` method for these types of property tags. So we skip the internal dictionary / KeyValuePair list creation of the slow path. I can effectively use the properties instead, so I'll do that.

Reply:
Well I'm doing that already 
![Uploading image.png…]()